pkgs/okta-aws-login/default.nix

```diff
@@ -1,29 +1,11 @@
-{ fetchurl, lib, stdenv }:
-
 let
-  version = "1.3.2";
-  urlTemplate = arch: "https://github.com/EarnestResearch/okta-aws-login/releases/download/v${version}/okta-aws-login-${arch}";
-  sources = {
-    x86_64-linux = fetchurl {
-      url = urlTemplate "Linux-x86_64";
-      sha256 = "13fdixjv2x9ja4csflhij7namgbc7g0zjmzcy4p82qy5m7sxbh9v";
-    };
-    x86_64-darwin = fetchurl {
-      url = urlTemplate "Darwin-x86_64";
-      sha256 = "081mkzmzv6790rwnqprkwhfmrwc3vx3hs30r1pqx166667nzhwiw";
-    };
-  };
-  unsupported = throw "Platform ${stdenv.hostPlatform.system} unsupported for okta-aws-login";
-in
-stdenv.mkDerivation rec {
-  name = "okta-aws-login-${version}";
-  src = lib.attrByPath [ stdenv.hostPlatform.system ] unsupported sources;
+  oktaAwsLogin = import (
+    builtins.fetchGit {
+      url = "https://github.com/EarnestResearch/okta-aws-login.git";
+      ref = "refs/heads/master";
+      rev = "9bedc99e8abf7fab22738c5e2ccc8935b6a9f391";
```

Comment:
This would be a good candidate to add as a niv source.  We already have the niv plumbing in this repo.

Reply:
Ah! Cool, thanks, will take a look.

Reply:
```sh
niv add okta-aws-login -o EarnestResearch -r okta-aws-login`
```

And then it'll be part of `nix/sources.nix`, which there are other examples of in the repo.